sdk/typescript/src/client/client.ts

```diff
@@ -536,15 +537,27 @@ export class IotaClient {
     }
 
     /**
-     * Return the latest system state content.
+     * Return the latest IOTA system state object on networks supporting protocol version `<= 4`.
```

Comment:
To match exactly what the rpc json has:
```suggestion
     * Return the latest IOTA system state object on networks supporting protocol version `< 5`.
```